localisation/00_memewc_united_localisation.csv

```diff
@@ -1,6 +1,18 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
 RULE_GROUP_WARCRAFT_ARTIFACTS;WarCraft - Artifacts;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT;�RWarCraft�!: Class Requirements for Physical Weapons;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only physical classes can use some physical weapons.;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT_UNRESTRICTED_DESC;Adult character of any class can use some physical weapons.;;;;;;;;;;;;;x
+RULE_RANGED_WEAPON_CLASS_REQUIREMENT;�RWarCraft�!: Class Requirements for Ranged Weapons;;;;;;;;;;;;;x
+RULE_RANGED_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only physical classes can use some ranged weapons.;;;;;;;;;;;;;x
+RULE_RANGED_WEAPON_CLASS_REQUIREMENT_UNRESTRICTED_DESC;Adult character of any class can use some ranged weapons.;;;;;;;;;;;;;x
+RULE_RANGED_WEAPON_CLASS_REQUIREMENT_RESTRICTED_DESC;Only ranged classes can use some ranged weapons.;;;;;;;;;;;;;x
+RULE_MAGICAL_WEAPON_CLASS_REQUIREMENT;�RWarCraft�!: Class Requirements for Magical Weapons;;;;;;;;;;;;;x
+RULE_MAGICAL_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only magical and combined classes can use some magical weapons.;;;;;;;;;;;;;x
```

Comment:
Maybe hybrid or multiple classes would work here better than combined classes? I'm not entirely sure if you are trying to say classes that can use both magic and weapons, i.e. Paladins, or a character who is both a mage and a warrior. Removed `some` because I don't _think_ its necessary. 
```suggestion
RULE_MAGICAL_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only magical and hybrid classes can use magical weapons.;;;;;;;;;;;;;x
```

Reply:
It's Palandins and Death Knights